CODEOWNERS

```diff
@@ -36,8 +37,8 @@ go.work.sum
 /internal/civisibility          @DataDog/ci-app-libraries @DataDog/apm-go
 
 # Gitlab configuration
-.gitlab-ci.yml                  @DataDog/dd-trace-go-guild @DataDog/apm-core-reliability-and-performance @DataDog/apm-go
-/.gitlab-ci                     @DataDog/dd-trace-go-guild @DataDog/apm-core-reliability-and-performance @DataDog/apm-go
```

Comment:
In the latest main the name of the 2nd team was fixed (changed to `@DataDog/apm-ecosystems-reliability`), not removed.